recipe/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+cargo build --release
+cargo build --release --manifest-path crates/c-api/Cargo.toml
+
+mkdir -p ${PREFIX}/bin
+mkdir -p ${PREFIX}/lib
+
+cp ${SRC_DIR}/target/release/wasmtime ${PREFIX}/bin/
+
+if [ `uname` == Darwin ]; then  
+    cp ${SRC_DIR}/target/release/libwasmtime.dylib ${PREFIX}/lib/
```

Comment:
We could potentially use `$SHLIB_EXT` to get the extension.

Reply:
yes